pom.xml

```diff
@@ -90,6 +90,7 @@ under the License.
 
   <properties>
     <mavenVersion>3.1.1</mavenVersion>
+    <mavenPluginToolsVersion>3.6.2-SNAPSHOT</mavenPluginToolsVersion>
```

Comment:
Please update to the upcoming 3.6.4.

Reply:
Set to 3.6.3 and resolved POM conflict, waiting for 3.6.4

Reply:
Hm, master already has all these changes plus m-p-p set to 3.6.2.... is this PR then only about "up m-p-p to 3.6.4" (once released)?

Reply:
m-p-p 3.6.4 was released 😄